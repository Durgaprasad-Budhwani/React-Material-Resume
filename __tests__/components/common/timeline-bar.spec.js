/**
 * Created by Durgaprasad Budhwani on 10/9/2017.
 */
/* global expect,it */
import React from 'react';
import TimelineBar from '../../../src/components/common/timeline-bar';
import renderer from 'react-test-renderer';

it('renders bubble', () => {
    const tree = renderer.create(<TimelineBar top={10} height={10} />).toJSON();
    expect(tree).toMatchSnapshot();
});
