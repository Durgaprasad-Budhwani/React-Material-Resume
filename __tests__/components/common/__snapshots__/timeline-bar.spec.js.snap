// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders bubble 1`] = `
<div
  className="TimelineBar-bar-1"
  style={
    Object {
      "height": 10,
      "top": 10,
    }
  }
/>
`;
